modules/java/native/java_machine.c

```diff
@@ -191,6 +191,9 @@ _setup_jvm_options_array(JavaVMSingleton *self, const gchar *jvm_options_str)
   jvm_options_array = _jvm_options_array_append(jvm_options_array,
                                                 g_strdup_printf("-Djava.system.class.loader=org.syslog_ng.SyslogNgClassLoader"));
 
+  jvm_options_array = _jvm_options_array_append(jvm_options_array,
```

Comment:
This should be added to the `predefined_options` as well.

Reply:
Fixed, thank you.